test/book-nav-stub.ts

```diff
@@ -0,0 +1,35 @@
+import { ModalManagerInterface } from '@internetarchive/modal-manager';
+import { MetadataResponse } from '@internetarchive/search-service';
+import { SharedResizeObserver } from '@internetarchive/shared-resize-observer';
+import { html, customElement, LitElement, property } from 'lit-element';
+
+@customElement('book-navigator')
+export class BookNavigator extends LitElement {
```

Comment:
Suggestion: Have this implement the `BookNavigator` interface since it looks like they're supposed to be the same, right?

Reply:
omg! i can't believe i missed this one. thanks again